src/test/java/org/programmers/springboardjpa/post/service/PostDefaultServiceStubTest.java

```diff
@@ -0,0 +1,100 @@
+package org.programmers.springboardjpa.post.service;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.programmers.springboardjpa.domain.post.dto.PostRequest;
+import org.programmers.springboardjpa.domain.post.dto.PostResponse;
+import org.programmers.springboardjpa.domain.post.repository.PostRepository;
+import org.programmers.springboardjpa.domain.post.service.PostConverter;
+import org.programmers.springboardjpa.domain.post.service.PostService;
+import org.programmers.springboardjpa.domain.user.dto.UserDto;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.domain.Pageable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+class PostDefaultServiceStubTest {
+
+    @Autowired
+    PostRepository postRepository;
+
+    @Autowired
+    PostConverter postConverter;
+
+    @Autowired
+    PostService postService;
+
+    PostResponse.PostResponseDto savedPost;
+
+    PostRequest.PostCreateRequestDto postCreateRequest;
+
+    @BeforeEach
+    void setUp() {
+        postCreateRequest = new PostRequest.PostCreateRequestDto(
+                "아이유는", "어느집 아이유", new UserDto.UserRequest(
+                "이상혁", 15,  "LOL"));
+        savedPost = postService.savePost(postCreateRequest);
+    }
+
+    @AfterEach
+    void tearDown() {
+        postRepository.deleteAll();
+    }
+
+    @Test
+    @DisplayName("id값이 자동 생성 전략에 따라 할당 되어 null이 아님을 확인 할 수 있다.")
+    void savePost() {
+        //then
+        assertThat(savedPost.id()).isNotNull();
+    }
+
+    @Test
+    @DisplayName("저장된 게시글의 아이디 값으로 찾아온 게시글의 아이디 값을 비교하여 동일한 게시글을 가져 올 수 있다.")
+    void findPost() {
+        //given
+        var foundPost = postService.getPost(savedPost.id());
+
+        //then
+        assertThat(foundPost.id()).isEqualTo(savedPost.id());
+    }
+
+    @Test
+    @DisplayName("")
+    void getAllPost() {
+        //given
+        var post = postService.getPost(savedPost.id());
+        List<PostResponse.PostResponseDto> postResponseList = new ArrayList<>();
+        postResponseList.add(post);
+
+        //when
+        var postList = postService.getPostList(
+                Pageable.ofSize(1).withPage(1));
```

Comment:
```suggestion
        List<PostResponse.PostResponseDto> postList = postService.getPostList(PageRequest.of(0, 1));
```

저는 보통 PageRequest.of(0, 1) PageRequest 클래스에서 제공하는 팩토리 메서드를 사용하는데 Pageable.ofSize(1).withPage(1)의 의미는 뭘까요?

Reply:
Pageable에 대한 구현체를 찾아보다 of 메소드를 통해 PageableImpl이라는 구현체를 통해 반환이 가능하여 테스트를 진행하려고 했었는데 PageRequest를 사용하는 방법이 맞는 것 같네요.
이 부분 추가적으로 학습이 필요할 것 같습니다.